Yank/restraints.py

```diff
@@ -714,22 +787,24 @@ def _create_restraint_force(self, particle1, particle2):
         parameter_names, parameter_values = zip(*self._bond_parameters.items())
 
         # Create bond force and lambda_restraints parameter to control it.
-        force = openmm.CustomBondForce('lambda_restraints * ' + self._energy_function)
-        force.addGlobalParameter('lambda_restraints', 1.0)
+        if len(particles1) == 1 and len(particles2) == 1:
+            # CustomCentroidBondForce works only on 64bit platforms. When the
```

Comment:
Where in the docs does it say it only supports 64-bit platforms? And what is the exact error it throws if you use the wrong platform?